census/customestimator/trainer/task.py

```diff
@@ -200,32 +171,6 @@ def _experiment_fn(run_config, hparams):
   os.environ['TF_CPP_MIN_LOG_LEVEL'] = str(
       tf.logging.__dict__[args.verbosity] / 10)
 
-  # If job_dir_reuse is False then remove the job_dir if it exists
-  if not args.reuse_job_dir:
-    if tf.gfile.Exists(args.job_dir):
-      tf.gfile.DeleteRecursively(args.job_dir)
-      tf.logging.info("Deleted job_dir {} to avoid re-use".format(args.job_dir))
-    else:
-      tf.logging.info("No job_dir available to delete")
-  else:
-    tf.logging.info("Reusing job_dir {} if it exists".format(args.job_dir))
-
-
   # Run the training job
-  # learn_runner pulls configuration information from environment
-  # variables using tf.learn.RunConfig and uses this configuration
-  # to conditionally execute Experiment, or param server code
-  learn_runner.run(
-      generate_experiment_fn(
-          min_eval_frequency=args.min_eval_frequency,
```

Comment:
It's pretty important for performance that we don't lose this arg.

Reply:
Also, please check the other examples for the same thing.

Reply:
@rhaertel80 : [`tf.estimator.train_and_evaluate`](https://www.tensorflow.org/api_docs/python/tf/estimator/train_and_evaluate), [`tf.estimator.TrainSpec`](https://www.tensorflow.org/api_docs/python/tf/estimator/TrainSpec), [`tf.estimator.EvalSpec`](https://www.tensorflow.org/api_docs/python/tf/estimator/EvalSpec), and (as a hail mary) [`tf.estimator.Estimator`](https://www.tensorflow.org/api_docs/python/tf/estimator/Estimator) do not seem to expose this parameter.

Does this mean users should be sticking with `tf.contrib.learn.Experiment` and `tf.contrib.learn.learn_runner` on ML Engine?

Also, because curiosity, how does this affect performance?

Reply:
So the answer to above comment is little bit involved. So one can specify the https://www.tensorflow.org/api_docs/python/tf/estimator/EvalSpec#throttle_secs to supposedly get the same behavior but the code I modified right now uses FinalExporter is not invoking such a behavior aka what you would expect from the throttle_sec. I have also played with the LatestExporter and I still don't seem to get the behavior. 

@nkashy1 eval frequency is important because it can bog down the chief otherwise.

Reply:
And since in current code eval is done once it doesn't impact performance IMO but I see that we need to be able to show continuous eval use-case which like I said earlier I am not able to reproduce a behavior for.